### PR TITLE
Adding Canadian Req

### PR DESCRIPTION
**Pull request by hsaunders0788**
Adding the Canadian requisite for community advocate interns to the list of locations and links.

### CODE REVIEW

**Comment by mmcelaney** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -73,10 +73,14 @@ Locations:
 
 - IBM Thomas J. Watson Research Center (Yorktown Heights, New York, USA)
 - IBM Research — Almaden (San Jose, California, USA)
+- IBM Markham (Markham, Ontario, Candada)
```
Comment (bradley-holt) [inline README.md]: Typo on the spelling of "Canada"

Comment (bradley-holt) [reply README.md]: Let's just say "IBM Canada (Markham, Ontario, Canada)"

**Review by bradley-holt — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hsaunders0788) [reply README.md]: Done

**Review by bradley-holt — APPROVED** [review]

**Pull request merged**